frontend/app.js

```diff
@@ -0,0 +1,81 @@
+// Home screen component
+const HomeComponent = {
+  render: (params) => homeScreen(params)
+}
+
+// Edit Schema screen component
+const SchemaComponent = {
+  render: (params) => schemaReport(params)
```

Comment:
I didn't get any use of params here.Is it used anywhere?

Reply:
Agreed.. It is not used anywhere and hence I removed it